_common

```diff
@@ -31,13 +31,20 @@ job_ids() {
 # an error, and the calling command and line
 runcli() {
     local args=("$@")
-    local rc output
+    local rc output errorlog errstr=
     set +e
-    output=$("${args[@]}" 2>&1)
+    errorlog=$(mktemp -t runcli-errorlog-XXXX)
```

Comment:
Also you could change:
```
errorlog=$(mktemp -t runcli-errorlog-XXXX)
```
to 
```
+mktemp; errorlog=$temp
```
and that would make an auto-deleting temp file for you.

Reply:
@perlpunk indicated that you don't use bpan/bashplus in deployment.

I made this standalone version of +trap for you to use: https://gist.github.com/ingydotnet/15a44654776f1709f04e67b1d35a307c